recipes-bsp/u-boot/u-boot-script-toradex_2020.07.bb

```diff
@@ -18,7 +18,7 @@ do_configure[noexec] = "1"
 do_compile[noexec] = "1"
 
 do_mkimage() {
-    sed -e 's/@@KERNEL_BOOTCMD@@/${KERNEL_BOOTCMD}/' \
+    sed -e 's/@@KERNEL_BOOTCMD@@/${KERNEL_BOOTCMD}/;s/@@KERNEL_IMAGETYPE@@/${KERNEL_IMAGETYPE}/' \
         "${WORKDIR}/boot.cmd.in" > ${B}/boot.cmd
```

Comment:
I'd replace this with:
```
sed -e 's/@@KERNEL_BOOTCMD@@/${KERNEL_BOOTCMD}/` \
    -e 's/@@KERNEL_IMAGETYPE@@/${KERNEL_IMAGETYPE}/' \
```
as it's easier to read and clearly shows it has multiple replacements.

Reply:
Done.